docs/docs/experimentation/connect-a-warehouse.md

```
@@ -50,6 +50,49 @@ The current status is shown on the warehouse card in **Environment Settings > Wa
+    INDEX idx_identity identifier TYPE bloom_filter GRANULARITY 4,
+
+    CONSTRAINT environment_key_not_empty CHECK environment_key != '',
+    CONSTRAINT event_not_empty           CHECK event != '',
+    CONSTRAINT timestamp_sane            CHECK timestamp > toDateTime64('2020-01-01 00:00:00', 3)
+)
+ENGINE = MergeTree
+PARTITION BY toYYYYMMDD(timestamp)
+ORDER BY (environment_key, event, feature_name, timestamp, identifier);
+```
+
+2. Go to **Environment Settings > Warehouse**, select **ClickHouse** and enter your connection details: host, port (9440
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)